README.md

```diff
@@ -22,9 +22,7 @@ A common goal of cloud computing is to abstract away operational tasks from appl
 developer efforts can be focused on providing business value. One feature common to most applications is the need to
 monitor application and hardware metrics. On
 [Google Cloud Platform (GCP)](https://cloud.google.com/) the [Stackdriver](https://cloud.google.com/stackdriver/) suite
-of products addresses the need for monitoring and alerting. As some companies may already have a solution this is a
-demo using [Datadog](https://www.datadoghq.com/). We show you how to collect nginx metrics using your existing Datadog account.
-
+of products addresses the need for monitoring and alerting. Some companies may already have a third party solution, however, and this project demonstrates configuring the use of [Datadog](https://www.datadoghq.com/) in a [GKE](https://cloud.google.com/kubernetes-engine/) environment. We show you how to collect [nginx](https://www.nginx.com/) metrics and pipe them to your existing Datadog account.
```

Comment:
We should still call it Kubernetes Engine, not GKE.

Reply:
k